xml/wgl.xml

```diff
@@ -413,13 +413,13 @@ Registry at https://github.com/KhronosGroup/OpenGL-Registry
         <enum value="0x21A0"        name="WGL_TYPE_RGBA_FLOAT_ATI"/>
             <unused start="0x21A1"/>
         <enum value="0x21A2"        name="WGL_GPU_FASTEST_TARGET_GPUS_AMD"/>
-        <enum value="0x21A3"        name="WGL_GPU_RAM_AMD"/>
-        <enum value="0x21A4"        name="WGL_GPU_CLOCK_AMD"/>
-        <enum value="0x21A5"        name="WGL_GPU_NUM_PIPES_AMD"/>
+        <enum value="0x21A3"        name="WGL_GPU_RAM_AMD" group="GPUProperty"/>
+        <enum value="0x21A4"        name="WGL_GPU_CLOCK_AMD" group="GPUProperty"/>
+        <enum value="0x21A5"        name="WGL_GPU_NUM_PIPES_AMD" group="GPUProperty"/>
         <enum value="0x21A5"        name="WGL_TEXTURE_RECTANGLE_ATI" comment="Duplicates unrelated WGL_GPU_NUM_PIPES_AMD"/>
-        <enum value="0x21A6"        name="WGL_GPU_NUM_SIMD_AMD"/>
-        <enum value="0x21A7"        name="WGL_GPU_NUM_RB_AMD"/>
-        <enum value="0x21A8"        name="WGL_GPU_NUM_SPI_AMD"/>
+        <enum value="0x21A6"        name="WGL_GPU_NUM_SIMD_AMD" group="GPUProperty"/>
```

Comment:
What's our ruling on vendor-specific enum groups? In gl.xml this would've likely been `GPUPropertyAMD`. Related: #481

Reply:
I'm fine with making this enum vendor specific, but I think most enum groups in `gl.xml` do not have a vendor postfix, even if it's mostly an extension concept. For this group, if someone else made an extension it would likely not fit into the same enum groups so it might reduce future confusion (but also, gl, wgl, and glx aren't getting that many new changes to them and probably won't).

Then there is also the fact that almost all API in wgl comes in the form of extensions, so basically all groups would need a postfix. But maybe that is what we want.